src/main.rs

```diff
@@ -102,6 +104,7 @@ async fn action_delete(item: web::Json<EntryRequestDto>) -> HttpResponse {
 
     // Load entries from custom.conf
     let mut custom_entries = load_custom_dnsmasq_entries_from_file();
+    let custom_entries_original = custom_entries.clone();
```

Comment:
why no in own function

Reply:
It is not possible to abstract this code any further. At least not with my Rust knowledge as of today.

Reply:
ok